Library/Homebrew/extend/os/linux/formula.rb

```diff
@@ -13,6 +14,10 @@ def shared_library(name, version = nil)
     "#{name}.so#{suffix}"
   end
 
+  def rpath
+    "'$ORIGIN/../lib'"
```

Comment:
The single quotes here seem very suspicious to me. I've verified that the `RPATH` of `abseil` does get set correctly to `$ORIGIN/../lib` without single quotes, but I strongly suspect that it should be up to the caller to add the single quotes when needed.

Reply:
We can drop it if it's not needed. I was just concerned about the shell eating the `$`, so I put that in there. I don't quite know how much the shell interprets things when you make `system` calls in the install method, however.

Reply:
The form `system "foo bar baz"` does go through the shell and shell expansion, but the preferred form `system "foo", "bar", "baz"` does not go through the shell and shell expansion.

Reply:
Does it hurt though? I think waiting for the caller to make sure the `$` is not interpreted by the shell is a footgun. If there's a reason to give callers that, then we can do so, but I'm not sure there is.